public/app/features/canvas/runtime/scene.tsx

```diff
@@ -67,13 +67,23 @@ export class Scene {
 
   inlineEditingCallback?: () => void;
 
+  readonly editModeEnabled = new BehaviorSubject<boolean>(false);
+
   constructor(
     cfg: CanvasFrameOptions,
     enableEditing: boolean,
     showAdvancedTypes: boolean,
     public onSave: (cfg: CanvasFrameOptions) => void
   ) {
     this.root = this.load(cfg, enableEditing, showAdvancedTypes);
+
+    // TODO: Unsubscribe from this (cleanup?)
+    this.editModeEnabled.subscribe((open) => {
```

Comment:
```suggestion
    this.subscription = this.editModeEnabled.subscribe((open) => {
```
and then later in some shutdown...
````
this.subscription.unsubscribe()
```

:)